app/api_topologies.go

```diff
@@ -31,6 +139,52 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
+type byName []APITopologyDesc
```

Comment:
Can do.  I put it there to be near the functions, and typically we don't like to mix functions and type definitions.